src/util.c

```diff
@@ -1132,3 +1132,93 @@ int utilTest(int argc, char **argv, int flags) {
     return 0;
 }
 #endif
+
+
+/*
+ * Copyright (c) 1998, 2015 Todd C. Miller <millert@openbsd.org>
```

Comment:
the `#ifdef REDIS_TEST` is usually the last part of the file.
i don't like the fact we add non-test code below it.